tests/vm_test.go

```diff
@@ -1282,7 +1282,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 							return false
 						}, 10*time.Second, 1*time.Second).Should(BeTrue())
 					} else {
-						Eventually(Expect(launcherPod.Status.Phase).To(Equal(k8sv1.PodFailed)), 160*time.Second, 1*time.Second).Should(BeTrue())
+						Eventually(Expect(launcherPod.Status.Phase).To, 160*time.Second, 1*time.Second).WithArguments(Equal(k8sv1.PodFailed)).Should(BeTrue())
```

Comment:
Is this a common practice? (Expect inside an eventually)

Reply:
Quick github search says it's not very common (mostly kubevirt forks):
https://github.com/search?q=%22Eventually%28Expect%28%22&type=code

Reply:
I would just return the `launcherPod.Status.Phase` and compare it to the desired PodPhase

Reply:
OK @akalenyu, done. But I'll need LGTM again (-;